msg/ThrusterCommand.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp           # time since system start (microseconds)
+
+float32 x1                 # duty cycle of thruster-pair x1 [-1;1]
+float32 x2                 # duty cycle of thruster-pair x2 [-1;1]
+float32 y1                 # duty cycle of thruster-pair y1 [-1;1]
+float32 y2                 # duty cycle of thruster-pair y2 [-1;1]
```

Comment:
Doesn't that duplicate https://github.com/PX4/PX4-Autopilot/blob/253208fdd4d3b415ef9ce019d7b5475a3e103a8a/msg/ActuatorMotors.msg ?

Reply:
This si related to the thruster_controller module, that then publishes motor controls that get translated to actuator outputs. The x's and y's are relative to body frame, but this should probably be removed since we don't need this to be upstream.